core/main_test.ts

```diff
@@ -1072,9 +1079,9 @@ nodes:
           asPlainObject([
             {
               target: {
-                database: "defaultProject",
-                schema: "defaultDataset",
-                name: "dest"
+                database: "defaultProject_projectSuffix",
+                schema: "defaultDataset_datasetSuffix",
+                name: "tablePrefix_dest"
```

Comment:
In Line 1092 of this same file, the list of all possible targets is outlined, but this override isn't being applied there.

Do I need to add additional changes elsewhere so all targets are resolved correctly?

Reply:
Yes. Nice spot with this!